openslides_backend/action/actions/option/update.py

```diff
@@ -121,13 +121,13 @@ def _get_option_and_poll(
     def _handle_poll_option_data(
         self, instance: Dict[str, Any], poll: Dict[str, Any]
     ) -> None:
-        data = self._get_data(instance)
         if poll.get("type") == "analog":
+            data = self._get_data(instance)
             pollmethod = poll["pollmethod"]
             for letter, key in option_keys_map.items():
                 if letter in pollmethod:
                     instance[key] = data.get(key, "-2.000000")
-                else:
+                elif data.get(key):
```

Comment:
Why?

Reply:
Because with value, this poll.option is not allowed and should raise the following exception. Without beeing used in data (data.get(key) returns None) or beeing set to None the the exception shouldn't be raised.
Without these change, a pollmethod missing one of the letters "YNA" ALWAYS raise the exception, because for example the "A" is not in the pollmethod, but in the list of letters.

Reply:
If "Y" is not in pollmethod, providing "Y" in the payload should raise an **exception**. If "Y": None it is an client error,